src/api/app/assets/javascripts/webui2/datatables.js

```diff
@@ -8,6 +8,8 @@
 function initializeDataTable(cssSelector, params){ // jshint ignore:line
   var defaultParams = {
```

Comment:
I think we should add default params to a variable that both methods share, as initializeTomoteDatatable is just extending them. This way we avoid repeating code and forgetting it in one of them (as has happened here) :wink:

Reply:
@Ana06 Changed it:smiley: